### DISCUSSION TITLE
my api data is not displaying

### DISCUSSION THREAD

**Discussion by JoyWanjiku** [mantinedev/mantine#3213] [Q&A]
![image](https://user-images.githubusercontent.com/76406254/208501084-3c1fa8ec-f1cb-48e6-a70b-98dbcdaadbce.png)
![image](https://user-images.githubusercontent.com/76406254/208501102-26995fc9-5c1a-4fa7-8dee-85358a722c7f.png)
i must be missing something very tiny right?
when i run console.log{patients} the data shows up and on the site i noticed the number of rows is equivalant to the number of data inputs stored in the api. so why wont it display?

**Comment by rtivital** [comment]
We do not provide DataTable component, if you have issues with it, you should consult the maintainer of the library.
